src/Modal/Modal.tsx

```diff
@@ -95,15 +99,19 @@ const Modal = memo(
                                     </button>
                                 </div>
                                 <div className={fr.cx("fr-modal__content")}>
-                                    <h1 id={titleId} className={fr.cx("fr-modal__title")}>
+                                    <TitleTag
+                                        id={titleId}
+                                        className={fr.cx("fr-modal__title")}
```

Comment:
Hello @maxgfr,  

I know this is anoying but you don't want to "swallow the class" in case it's been provided in the props.


```suggestion
                                        id={titleId}
                                        {...titleProps]
                                        className={cx(titleProps.className,fr.cx("fr-modal__title"))}
                                        
```

Reply:
Thanks !